package.json

```diff
@@ -13,7 +13,7 @@
     "format:prettier": "prettier \"**/*.{cjs,html,js,json,md,ts}\" --write",
     "ignore-sync": "ignore-sync .",
     "lint": "eslint \"**/*.{js,ts}\"",
-    "nuke": "rm -rf node_modules packages/{*,labs/*}/node_modules && npm install && npm run bootstrap && npm run clean",
+    "nuke": "rm -rf node_modules package-lock.json && npm install && npx lerna exec 'rm -rf node_modules package-lock.json' && npx lerna bootstrap && npm run clean",
```

Comment:
I think `npx` is unnecessary in an npm script

Reply:
true fixed